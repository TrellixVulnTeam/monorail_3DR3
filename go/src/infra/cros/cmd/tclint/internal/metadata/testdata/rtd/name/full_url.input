{
    "remoteTestDrivers": [
        {
            "name": "https://user@host.org/remoteTestDriver/tauto"
        }
    ]
}