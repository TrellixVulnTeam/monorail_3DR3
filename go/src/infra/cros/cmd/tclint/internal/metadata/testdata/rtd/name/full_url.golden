[
"## Errors ##",
"RemoteTestDriver 'https://user@host.org/remoteTestDriver/tauto': name must be a URL path component (https://aip.dev/122), found non-empty host 'host.org'",
"RemoteTestDriver 'https://user@host.org/remoteTestDriver/tauto': name must be a URL path component (https://aip.dev/122), found non-empty scheme 'https'",
"RemoteTestDriver 'https://user@host.org/remoteTestDriver/tauto': name must be a URL path component (https://aip.dev/122), found non-empty user information 'user'",
"RemoteTestDriver 'https://user@host.org/remoteTestDriver/tauto': name must be a URL relative path component (https://aip.dev/122), found absolute path '/remoteTestDriver/tauto'"
]